web/src/components/questions/Questions.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) [2022-2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import GenericQuestion from "~/components/questions/GenericQuestion";
+import QuestionWithPassword from "~/components/questions/QuestionWithPassword";
+import LuksActivationQuestion from "~/components/questions/LuksActivationQuestion";
```

Comment:
This three import also reminds me the article I shared on slack channel two weeks ago about naming: https://kyleshevlin.com/prefer-noun-adjective-naming

It isn't exactly the same case than the article, but maybe going for `QuestionGeneric`, `QuestionWithPassword`, and `QuestionLuksActivation` would make them more consistent from a naming POV. Again, no strong opinion.

Reply:
Or maybe we could just drop the `Question` prefix/suffix since they are already in the `questions` namespace.